companyweb_base/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="1">
+    <record id="companyweb_alacarte" model="ir.config_parameter">
+        <field name="key">companyweb.alacarte</field>
+        <field
+            name="value"
+        >https://connect.companyweb.be/V1.3/alacarteservice.asmx</field>
+    </record>
+    <!-- Companyweb is ok with those key being visible on github -->
+    <record id="service_integrator_id" model="ir.config_parameter">
+        <field name="key">service.integrator.id</field>
+        <field name="value">acsone</field>
+    </record>
+    <record id="service_integrator_secret" model="ir.config_parameter">
+        <field name="key">service.integrator.secret</field>
+        <field name="value">ECAB8ACF-9AE1-4E90-BD0D-05A1F47A3FE9</field>
+    </record>
+</odoo>
```

Comment:
The service integrator id and secret is not meant to be changed by users or administrator. So I think we should keep it in the code.